services/src/main/java/org/keycloak/protocol/oidc/par/endpoints/ParEndpoint.java

```diff
@@ -57,6 +61,7 @@
 public class ParEndpoint extends AbstractParEndpoint {
 
     public static final String PAR_CREATED_TIME = "par.created.time";
+    public static final String PAR_DPOP_PROF_JKT = "par.dpop.prof.jkt";
```

Comment:
Consider keeping spelling `PROOF` instead of `PROF` to align with `OAuthErrorException.INVALID_DPOP_PROOF` for example.
```suggestion
    public static final String PAR_DPOP_PROOF_JKT = "par.dpop.proof.jkt";
```

Reply:
+1 to this suggestion

Reply:
@IngridPuppet @mposolda Thank you. I fixed the point as you suggested.